packages/system/src/Item/Components/ItemSearchInput/ItemSearchInput.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Story } from '@storybook/react';
+import { ItemSearchInput } from './ItemSearchInput';
+import { Item } from '@openmsupply-client/common';
+
+export default {
+  title: 'Item/ItemSearchInput',
```

Comment:
might be better under `Inputs` - we will end up with a lot of stories I expect, and grouping them nicely is going to be a problem.

Reply:
Heh I had the same thought and my version of grouping nicely was to put it under the type 🤣 

I do think the story book will likely get massive. I figured if someone was to look for a component they could use to search for items or names, looking under a names or items header. Though at the same time if someone was looking for a component they could copy to create an autocomplete for looking up periods, then they might look at inputs.

I'm not sure, but I can also see `inputs` become massive in itself and needing to be separated out. I'd like to keep it like this for now? It's just changing the string from `Item` to `Inputs` so should be an easy change when it's more obvious you're right? 😁